staging/src/k8s.io/apimachinery/pkg/util/validation/validation_test.go

```diff
@@ -238,7 +239,7 @@ func TestIsQualifiedName(t *testing.T) {
 		"example.com/Uppercase_Is_OK_123",
 		"requests.storage-foo",
 		strings.Repeat("a", 63),
-		strings.Repeat("a", 253) + "/" + strings.Repeat("b", 63),
```

Comment:
I am considering add a logic, if the subdomain only have one part, we  can consider it's max length is 253, others, if have more than one parts, every part should be no more than 63, does this make sense ?  this will bypass this case, and tighten the some kind validation.